openmetadata-ui/src/main/resources/ui/src/utils/UserDataUtils.ts

```diff
@@ -67,9 +67,12 @@ export const getUserDataFromOidc = (
   const images = oidcUser.profile.picture
     ? getImages(oidcUser.profile.picture)
     : undefined;
+  const profileEmail = oidcUser.profile.email;
+  const email = isEmpty(profileEmail) ? userData.email : profileEmail;
```

Comment:
@vivekratnavel,  Can't this be like this?
If `profileEmail` is an empty string, the ternary will consider it as a false value, then I think we don't need the `isEmpty` method there.

```suggestion
  const email = profileEmail ? userData.email : profileEmail;
```